packages/unraid-api-plugin-connect/src/connect/connect-settings.service.ts

```
@@ -237,11 +221,11 @@ export class ConnectSettingsService {
      * @returns true if the mode was changed, false otherwise
      */
     private async setSandboxMode(sandboxEnabled: boolean): Promise<boolean> {
-        const { store, getters } = await import('@app/store/index.js');
-        const currentSandbox = getters.config().local.sandbox;
+        throw new Error('Not implemented');
+        const currentSandbox = this.configService.get('store.local.sandbox');
         const sandbox = sandboxEnabled ? 'yes' : 'no';
         if (currentSandbox === sandbox) return false;
-        store.dispatch(updateUserConfig({ local: { sandbox } }));
+        this.configService.set('store.local.sandbox', sandbox);
         return true;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)